components/observations/SimpleForm.tsx

```diff
@@ -246,11 +246,7 @@ export const SimpleForm: React.FC<{
                           multiline: true,
                         }}
                       />
-<<<<<<< HEAD
-                      <LocationField name="location_point" label="Latitude/Longitude" />
-=======
                       <LocationField name="mapLocation" label="Latitude/Longitude" center={center_id} />
```

Comment:
this is wrong, should be `location_point`

Reply:
Oops, fixed